### PR TITLE
feat: Add Web Manifest, App Icons and Favicon

### PR DESCRIPTION
**Pull request by RubenSibon**
- Includes NPM script to shell script in @amsterdam/design-system-assets module.
- Includes NPM script to clean directory.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +23,8 @@
     "unlink": "npm unlink @amsterdam/design-system-assets @amsterdam/design-system-css @amsterdam/design-system-react @amsterdam/design-system-react-icons @amsterdam/design-system-tokens",
     "update": "npm-check-updates --configFileName .ncurc.major.js --deep --dep dev,prod --target latest --upgrade && npm install",
     "update:minor": "npm-check-updates --configFileName .ncurc.minor.js --deep --dep dev,prod --target minor --upgrade && npm install",
-    "update:patch": "npm-check-updates --configFileName .ncurc.patch.js --deep --dep dev,prod --target patch --upgrade && npm install"
+    "update:patch": "npm-check-updates --configFileName .ncurc.patch.js --deep --dep dev,prod --target patch --upgrade && npm install",
+    "clean": "rm -r node_modules dist"
```
[the remark this answers is not in the document]
Comment (RubenSibon) [reply package.json:27]: I did not know about `rimraf`. I've been reading into it, but this article made me decide to remove the clean option altogether.

And I really wonder how many JavaScript developers who are on Windows are not using a *sh shell. It seems impossible to find any statistics on that sadly.

File: public/app-icons [hunk 2]
```diff
@@ -0,0 +1 @@
+../node_modules/@amsterdam/design-system-assets/app-icons
```
Comment (alimpens) [inline public/app-icons:1]: This isn't released yet, so it's not going to work in this repo, right?

Comment (RubenSibon) [reply public/app-icons:1]: True I guess? I don't know when we do releases. When does that happen?

Comment (alimpens) [reply public/app-icons:1]: Probably this week, barring any unforseen circumstances :D 

I'll turn this PR into a draft for now

**Comment by VincentSmedinga** [comment]
@RubenSibon Can you please merge the `main` branch and mark the PR as ready?

**Comment by RubenSibon** [comment]
Tested locally. Let's check the deployment as well. With Firefox' Dev Tools you can inspect the Web Manifest.

**Comment by RubenSibon** [comment]
On `npm run build` the assets are correctly compiled into `out` dir.

File: package.json [hunk 3]
```diff
@@ -21,7 +26,7 @@
     "update:patch": "npm-check-updates --configFileName .ncurc.patch.js --deep --dep dev,prod --target patch --upgrade && npm install"
   },
   "dependencies": {
```
Comment (VincentSmedinga) [inline package.json:28]: Why did you update only this one package? All but the `react` package of the `@amsterdam` dependencies can be removed if we update to 0.9.0 immediately. The dependency on `@babel\runtime` can then be removed as well.

**Pull request merged**